tutorials/rise_mnist.ipynb

```diff
@@ -5,14 +5,13 @@
    "id": "hispanic-instrumentation",
```

Comment:
<p>See my comment about 'fraction of the image' not being accurate in the other notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Corrected!